src/tilda.h

```diff
@@ -34,6 +34,27 @@ struct lock_info
     gint file_descriptor;
 };
 
+struct tilda_cli_opts {
```

Comment:
Please name this `tilda_cli_options`. Please also typedef this, so you can drop the `struct` keyword.